src/main/java/com/kodilla/ecommercee/controller/CartController.java

```diff
@@ -1,4 +1,36 @@
 package com.kodilla.ecommercee.controller;
 
+import com.kodilla.ecommercee.dto.CartDto;
+import com.kodilla.ecommercee.dto.OrderDto;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+
+@CrossOrigin(origins = "*")
+@RestController
+@RequestMapping("v1/ecommercee/cart")
 public class CartController {
-}
+
+    @PostMapping(value = "createCart")
+    public  void createCart(@RequestParam long userId) {
+    }
+
+    @GetMapping(value = "getCart")
+    public CartDto getCart(@RequestParam long userId) {
+        return new CartDto(new ArrayList<>());
+    }
+
+    @PostMapping(value = "addItem")
+    public  void addItemToCard(@PathVariable long userId, @PathVariable long itemId) {
```

Comment:
nie znam adnotacji @PathVariable, ale zawsze gdy chcemy przekazać parametr na końcu adresu url , uzywalismy @RequestParam . Nie zapomniałeś tego zmienić z poprzedniej wersji controlera ? 
Jeśli @PathVariable działa to sorry za zamieszanie.